charts/dawarich/templates/deployment.yaml

```diff
@@ -32,6 +32,7 @@ spec:
         {{- toYaml .Values.podSecurityContext | nindent 8 }}
       initContainers:
         {{ include "dawarich.initContainers" . | nindent 8 }}
+        {{- if .Values.postgresql.enabled }}
```

Comment:
Should we somehow remind the chart‘s user that they either have to enable PostGIS themselves, or grant dawarich superuser rights?